<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormPesanan</class>
 <widget class="QWidget" name="FormPesanan">
  <property name="geometry"> <rect> <x>0</x> <y>0</y> <width>750</width> <height>600</height> </rect> </property>
  <property name="windowTitle"> <string>Data Pesanan</string> </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry"> <rect> <x>50</x> <y>20</y> <width>450</width> <height>320</height> </rect> </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0"><widget class="QLabel" name="l1"><property name="text"><string>ID Pesanan</string></property></widget></item>
    <item row="0" column="1"><widget class="QLineEdit" name="editID"/></item>
    <item row="1" column="0"><widget class="QLabel" name="l2"><property name="text"><string>ID Marketing</string></property></widget></item>
    <item row="1" column="1"><widget class="QLineEdit" name="editIDMkt"/></item>
    <item row="2" column="0"><widget class="QLabel" name="l3"><property name="text"><string>ID Staf</string></property></widget></item>
    <item row="2" column="1"><widget class="QLineEdit" name="editIDStaf"/></item>
    <item row="3" column="0"><widget class="QLabel" name="l4"><property name="text"><string>ID Barang</string></property></widget></item>
    <item row="3" column="1"><widget class="QLineEdit" name="editIDBrg"/></item>
    <item row="4" column="0"><widget class="QLabel" name="l5"><property name="text"><string>Permintaan</string></property></widget></item>
    <item row="4" column="1"><widget class="QLineEdit" name="editMinta"/></item>
    <item row="5" column="0"><widget class="QLabel" name="l6"><property name="text"><string>Pengiriman</string></property></widget></item>
    <item row="5" column="1"><widget class="QLineEdit" name="editKirim"/></item>
    <item row="6" column="0"><widget class="QLabel" name="l7"><property name="text"><string>Tgl Minta</string></property></widget></item>
    <item row="6" column="1"><widget class="QLineEdit" name="editTglMinta"/></item>
    <item row="7" column="0"><widget class="QLabel" name="l8"><property name="text"><string>Tgl Kirim</string></property></widget></item>
    <item row="7" column="1"><widget class="QLineEdit" name="editTglKirim"/></item>
    <item row="8" column="0"><widget class="QLabel" name="l9"><property name="text"><string>Status</string></property></widget></item>
    <item row="8" column="1"><widget class="QLineEdit" name="editStatus"/></item>
    <item row="9" column="0"><widget class="QLabel" name="l10"><property name="text"><string>Rekomendasi</string></property></widget></item>
    <item row="9" column="1"><widget class="QLineEdit" name="editRek"/></item>
    <item row="10" column="0"><widget class="QLabel" name="l11"><property name="text"><string>Status Rek</string></property></widget></item>
    <item row="10" column="1"><widget class="QLineEdit" name="editStatRek"/></item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnSimpan"> <property name="geometry"> <rect> <x>60</x> <y>350</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Simpan</string> </property> </widget>
  <widget class="QPushButton" name="btnHapus"> <property name="geometry"> <rect> <x>170</x> <y>350</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Hapus</string> </property> </widget>
  <widget class="QPushButton" name="btnUbah"> <property name="geometry"> <rect> <x>280</x> <y>350</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Ubah</string> </property> </widget>
  <widget class="QPushButton" name="btnBersih"> <property name="geometry"> <rect> <x>390</x> <y>350</y> <width>90</width> <height>29</height> </rect> </property> <property name="text"> <string>Bersih</string> </property> </widget>
  <widget class="QTableWidget" name="tabelPesanan">
   <property name="geometry"> <rect> <x>50</x> <y>400</y> <width>650</width> <height>180</height> </rect> </property>
   <column><property name="text"><string>ID</string></property></column>
   <column><property name="text"><string>Barang</string></property></column>
   <column><property name="text"><string>Minta</string></property></column>
   <column><property name="text"><string>Kirim</string></property></column>
   <column><property name="text"><string>Status</string></property></column>
  </widget>
 </widget>
 <resources/> <connections/>
</ui>
